dist/common/scripts/node_exporter_install

```diff
@@ -65,7 +63,7 @@ if __name__ == '__main__':
             node_exporter_p.unlink()
         os.symlink('{install_dir}/node_exporter-{version}.linux-amd64/node_exporter'.format(install_dir=INSTALL_DIR, version=VERSION), str(node_exporter_p))
         if is_systemd():
-            node_exporter = systemd_unit('node-exporter.service')
+            node_exporter = SystemdUnit('node-exporter.service')
```

Comment:
And this?

Reply:
This is a class and they should be named in PascalCase, again PEP8

Reply:
Ok, I think you're applying pylint recommendations for PEP8, not adding pylint execution.

Reply:
hmm, I see no point of adding a test that would just fail... wouldn't it ?

Reply:
Please word the changelog message in a way that isn't confusing.

You wrote "Adding pylint execution to test.py"

I expected a change to test.py that executes pylint.

Either the wording is incorrect, or the change is incorrect (or both). 

My expectation is that the wording matches the change.

Reply:
I've change the description a bit to explain this change is also applying the pylint recommendation. (otherwise it would be failing)